README.md

```diff
@@ -110,136 +115,159 @@ gofmt -w ./pkg
 gofmt -w ./cmd 
 ```
 
-Run the tests with 
+### Run Unit Tests
 
 ```bash
 make test-unit
 ```
 
 It is suggested to run `integration` and `e2e` tests with CI.  This is automatic when opening a PR.
 
-### Run the Binary Locally 
 
-The operator expects the `IMAGE` env var to be set in order to know which console to deploy. In 
-addition, it is recommended to use the `--create-default-console` flag as this ensures the operator 
-will deploy a console even if a `cr.yaml` for the openshift console is not found.  The console will 
-be deployed with reasonable default values. 
+### Development Against a 4.0 Dev Cluster 
 
-The `./examples/config.yaml` file mirrors `04-config.yaml` and can be used for dev testing.  Currently 
-it only controls leader election and provides nothing interesting to configure.
+The approach here is to build & deploy your code in a new container on a development cluster.  Don't 
+be put off by the need to redeploy your container, this is intended to provide a quick feedback loop.
 
-Run:
+#### Create a Cluster
+To develop features for the `console-operator`, you will need to run your code against a dev cluster.
+The `console-operator` expects to be running in a container.  It is difficult to fake a local 
+environment, and the debuggin experience is not like debugging a real container.  Instead, do the following 
+to set yourself up to build your binary & deploy a new container quickly and frequently.
 
-```bash 
-IMAGE=docker.io/openshift/origin-console:latest \
-    console operator \
-    --kubeconfig $HOME/.kube/config \
-    --config examples/config.yaml \
-    --create-default-console \
-    --v 4
+Visit [https://try.openshift.com/](https://try.openshift.com/), download the installer and create 
+a cluster.  [Instructions](https://cloud.openshift.com/clusters/install) (including pull secret) 
+are maintained here.
+
+```bash
+# create a directory for your install config
+# aws is recommended
+mkdir ~/openshift/aws/us-east
+cd ~/openshift/aws/us-east
+# generate configs using the wizard
+openshift-install create install-config
+# then run the installer to get a cluster 
+openshift-install create cluster --dir ~/openshift/aws/us-east --log-level debug
 ```
 
-NOTE: your `--kubeconfig` may be in another location.
+If successful, you should have gotten instructions to set `KUBECONFIG`, login to the console, etc.
 
-### Deploy an alternative image 
+#### Shut down CVO & the Default Console Operator
```

Comment:
Shutting down CVO is essential.  You need to deploy your code w/o it automatically redeploying the release image of the `console-operator`.

Reply:
I contemplated adding a few alternatives:

```bash
# delete CVO, bwa hahahahahaha
oc delete namespace openshift-cluster-version
# scale down CVO, a little more sane...
oc scale deployment cluster-version-operator --replicas 0 --namespace openshift-cluster-version
```